<javaelement>
	<package>
		java.rmi.server
	</package>
	<type>
		<name>
			public class ServerCloneException  extends CloneNotSupportedException
		</name>
		<description>
			A ServerCloneException is thrown if a remote exception occurs during the cloning of a UnicastRemoteObject. As of release 1.4, this exception has been retrofitted to conform to the general purpose exception-chaining mechanism. The "nested exception" that may be provided at construction time and accessed via the public detail field is now known as the cause, and may be accessed via the Throwable.getCause() method, as well as the aforementioned "legacy field." Invoking the method Throwable.initCause(Throwable) on an instance of ServerCloneException always throws IllegalStateException.
		</description>
		<methods>
			<method>
				<name>
					public Exception detail
				</name>
				<description>
					The cause of the exception. This field predates the general-purpose exception chaining facility. The Throwable.getCause() method is now the preferred means of obtaining this information.
				</description>
				
			</method>
			<method>
				<name>
					public ServerCloneException(String s)
				</name>
				<description>
					Constructs a ServerCloneException with the specified detail message.
				</description>
				
			</method>
			<method>
				<name>
					public ServerCloneException(String s,                      Exception cause)
				</name>
				<description>
					Constructs a ServerCloneException with the specified detail message and cause.
				</description>
				
			</method>
			<method>
				<name>
					public String getMessage()
				</name>
				<description>
					Returns the detail message, including the message from the cause, if any, of this exception.
				</description>
				
			</method>
			<method>
				<name>
					public Throwable getCause()
				</name>
				<description>
					Returns the cause of this exception. This method returns the value of the detail field.
				</description>
				
			</method>
			
		</methods>
		
	</type>
	
</javaelement>
